google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracerTest.java

```diff
@@ -327,6 +329,51 @@ public void testReadRowsOperationLatenciesOnAuthorizedView() {
     assertThat(value).isIn(Range.closed(SERVER_LATENCY, elapsed));
   }
 
+  @Test
+  public void testFirstResponseLatencies() {
+    Stopwatch firstResponseTimer = Stopwatch.createStarted();
+    stub.readRowsCallable()
+        .call(
+            Query.create(FIRST_RESPONSE_TABLE_ID),
+            new ResponseObserver<Row>() {
+              @Override
+              public void onStart(StreamController controller) {}
+
+              @Override
+              public void onResponse(Row response) {
+                if (firstResponseTimer.isRunning()) {
+                  firstResponseTimer.stop();
+                }
+                try {
```

Comment:
This is pretty fragile - you are assuming that operation timer will not stop on onComplete while the caller is blocking onResponse.

I think it would be much safer to send 2 responses with a delay

Reply:
Server is sending back 2 responses so I think this is fine? Added a comment